.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ jobs:
           command: |
             set -x
             cp .env.test .env
-            docker-compose build
+            docker-compose build db api frontend
```

Comment:
These were explicitly named so it doesn't build every container. This comes in to larger play on https://github.com/truetandem/e-QIP-prototype/pull/1966